src/components/Gesture.svelte

```diff
@@ -63,7 +88,8 @@
   }
 
   // method for recording data point for that specific gesture
-  function recordClicked(e?: Event): void {
+  async function recordClicked(e?: Event): Promise<void> {
```

Comment:
This function should probably be renamed now, as it is not called on click, but is being called after countdown finishes.

Maybe rename it simply to `recordData`? Also the `e` argument can be removed, as it is no longer going to be receiving an event from the button click.